Automattic/jetpack
# Contact Form: remove reCAPTCHA support

#### Changes proposed in this Pull Request:

Follow-up from #9389
Internal reference: p1HpG7-6mq-p2
More details: https://github.com/Automattic/jetpack/pull/9389#issuecomment-467976310

After further discussion, we will not be adding this feature to the contact form.
This reverts commit a3bebd3334ac86790b107a77d27493f898b1905e.

#### Testing instructions:

* Go to Posts > Add New
* Try to insert a contact form (using both the classic editor and the block editor)
* Make sure everything works as expected.

#### Proposed changelog entry for your changes:

* None
